content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -363,6 +363,38 @@ rules:
   verbs: ["get", "list", "watch"]
 ```
 
+#### Write Access for Endpoints
+
+Kubernetes clusters created before Kubernetes 1.22 include write access to
+Endpoints in the aggregated "edit" and "admin" roles. As a mitigation for
+[CVE-2021-25740](https://github.com/kubernetes/kubernetes/issues/103675), this
+access will not be part of the aggregated roles in newly created clusters using
+Kubernetes 1.22+.
+
+If you want new clusters to retain this level of access in the aggregated roles,
+you can create the following ClusterRole:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I would make this be a downloadable example that uses the `codenew` shortcode.

Reply:
Updated, thanks!